microsite/blog/2020-09-30-backstage-design-system.md

```diff
@@ -0,0 +1,108 @@
+
+Now that we’ve familiarized ourselves with the basic parts of a plugin, let’s take a look at some of the tools and resources that make designing plugins even easier.
+
+## Tools and resources
+
+The quickest way to start is by duplicating our [Figma UI kit](https://www.figma.com/community/file/850673348101741100), then clicking on the Create a Plugin page on the left-hand side to grab some templates. Feel free to experiment and play around! Figma is a fantastic, multiplayer tool that allows for designers and developers to co-create components and share specs with ease. We’re excited to see what you design and develop.
+
+As you’re designing and building your plugins, make sure to take advantage of these helpful resources.
+
+![img](assets/backstage-figma1-DS.png)
+![img](assets/backstage-figma2-DS.png)
+
+**[Figma Community](https://www.figma.com/@backstage)**
+
+We are thrilled to be using Figma Community to share our design assets. You can duplicate our UI kit and design your own plugin for Backstage. Figma Community is currently in beta, so this is a neat opportunity to be testing out a new way of working. At the moment, it doesn’t support auto-updating of duplicated files, so we’ll be announcing new versions of our UI kit on Discord in the #design channel.
+
+![img](assets/backstage-storybook-DS.png)
+
+**[Storybook](https://backstage.io/storybook/)**
+
+This is where you can view our reusable components. If you’d like to help build up our design system, you can also help us add components that we’ve designed to Storybook as well. We post newly designed components and patterns to GitHub as issues, where contributors can pick them up and add them to our Storybook so they’re available for the rest of the Backstage community.
+
+![img](assets/backstage-guidelines-DS.png)
+
+**[Guidelines](https://backstage.io/docs/dls/design)**
+
+To keep up with our latest design guidelines, go to [Designing for Backstage](https://backstage.io/docs/dls/design). You’ll find more how-to’s and you can also learn more about our design philosophy and practices there. Down the line, we plan on including more in-depth component rules in this section (i.e., dos/don’ts, use cases, etc.).
+
+![img](assets/backstage-github-DS.png)
+
+**[GitHub](https://github.com/spotify/backstage)**
+
+Join in on the action by submitting issues and opening pull requests for all things related to components and patterns in Backstage.
+
+![img](assets/backstage-discord-DS.png)
+
+**[Discord](https://discord.com/invite/MUpMjP2)**
+All design questions should be directed to the [#design](https://discord.com/channels/687207715902193673/696709358544879716) channel.
+
+## What's next for Backstage design
+
+![img](assets/backstage-guide-DS.png)
```

Comment:
This extra image shouldn't be there right? @katz95

Reply:
oh good catch!! yup, that shouldn't be there!